pkg/code/code_test.go

```diff
@@ -0,0 +1,52 @@
+package code
+
+import (
+	"testing"
+
+	"evylang.dev/evy/pkg/assert"
+)
+
+func TestMake(t *testing.T) {
```

Comment:
Make should ideally return an `error` when:
- there is no definition for the provided op
- an invalid operand width is given

This extra return makes compiler tests much more noisy - perhaps a MustMake  test helper that fails the test if an error is thrown?

Reply:
Seems like a good idea.